src/forward.jl

```diff
@@ -44,9 +44,9 @@ end
 """
    forwardrecurrence(N, A, B, C, x)
 
-evaluates the first `N` orthogonal polynomials at points `x`,
-where `A`, `B`, and `C` are `AbstractVector`s containing the first form recurrence coefficients,
-i.e. it returns 
+evaluates the first `N` orthogonal polynomials at point `x`,
+where `A`, `B`, and `C` are `AbstractVector`s containing the first form recurrence coefficients as defined in 
+[DLMF](https://dlmf.nist.gov/18.9), i.e. it returns Pᵢ(x) for i = 0, 1, ..., N-1
 """
 forwardrecurrence(N::Integer, A::AbstractVector, B::AbstractVector, C::AbstractVector, x) =
```

Comment:
Should the meaning of `N` change here? Currently it means compute the first `N` (including P0) vs compute up to PN. To me, the later makes more sense.

Reply:
Julia uses 1-based indexing so I think N specifying the length of the vector is more natural